### DISCUSSION TITLE
Edditing point feature - avoiding the button for single feature editation

### DISCUSSION THREAD

**Discussion by m-a-r-t-in** [opengisch/QField#3831] [Q&A]
Hi all,

Does anyone know if it is possible to avoid the necessity to switch edditation mode for an each individual feature when the layer is already in edditing mode ?
We would like to use QField for collection of large datasets of trees ( I am working in the field of forest ecology). When edditing houndreds of point features a day every step matters, enabling edditation for every single feature seems kind of useless, considering the layer is already switched into edditing mode.

Thanks forn any help or suggestions.

**Comment by Bandorai** [comment]
Hi M-a-r-t-in,
I solve the same problem. Till now not succesfully. If anyone knows, it would really help me.
Thanks

**Comment by BoswachterMarc** [comment]
Once I am in editing mode, I can start collecting data (in different layers) without having to enable it for each consecutive feature. I think this is common practice. There is even a way to remember attribure values, very handy for large (tree) datasets.

See https://docs.qfield.org/how-to/digitize/

**Comment by Bandorai** [comment]
Thanks Marc for very helpful point about remembering attributes. I'll use it.
But I still don't know, how to avoid the unnecessary button for editing attributes of selected feature (looks like letter "A" with pen, next to button for editing geometry). I'd like to edit attributes directly. The main part of my work is about updating attributes whereas geometry remains unchanged.
If anyone has any idea, thanks in advance.
